x-pack/plugin/eql/src/test/java/org/elasticsearch/xpack/eql/expression/function/scalar/string/StringUtilsTests.java

```diff
@@ -8,10 +8,21 @@
 
 import org.elasticsearch.test.ESTestCase;
 
+import java.util.concurrent.Callable;
+
+import static org.elasticsearch.xpack.eql.expression.function.scalar.string.StringUtils.stringContains;
 import static org.elasticsearch.xpack.eql.expression.function.scalar.string.StringUtils.substringSlice;
 
 public class StringUtilsTests extends ESTestCase {
 
+    protected static final int NUMBER_OF_TEST_RUNS = 20;
+
+    private static void run(Callable<Void> callable) throws Exception {
+        for (int runs = 0; runs < NUMBER_OF_TEST_RUNS; runs++) {
```

Comment:
Imo, I don't think we need this high degree of unit tests for this particular functionality, but wait for @costin's take on this before modifying this or leaving it as is.